### PR TITLE
Take-home interview submission for Raleigh Littles

### PR DESCRIPTION
**Pull request by raleighlittles**
See SETUP.md for info on how to set up, as well as a small demo on usage. I tested this on Windows 10 so I can't guarantee the setup instructions would work on other operating systems. 

Any and all feedback on my code would be appreciated -- thanks in advance.

### CODE REVIEW

File: db/schema.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+drop table if exists planets;
+create table planets (
+planet_id integer primary key autoincrement,
+ordinality int,
+planet_name text,
+size real,
+distance real,
+description text,
+constraint unique_ordinality UNIQUE (ordinality),
+constraint unique_planet_name UNIQUE (planet_name),
```
Comment (markBowman) [inline db/schema.sql:11]: The extra comma at end of columns breaks this schema for me.

File: initial.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+@app.route('/input', methods=['GET'])
+def input():
+    return flask.render_template("input.html")
+
+
+@app.route('/detail/<int:planet_id>', methods=['GET'])
+def get_planet(planet_id):
+    query = "SELECT planet_name, size, distance, ordinality, description FROM planets WHERE planet_id = {}".format(
+        str(planet_id))
+    flask.render_template('detail.html', planet=query_db(query, one=True))
```
Comment (markBowman) [inline initial.py:104]: Missing return

**Review by markBowman — COMMENTED** [review]
Hi Raleigh, Thank you for making the time to do our coding challenge. Your solutions looks great (with a couple of minor tweaks).
We will be in contact shortly with further details.
Thanks again,
Mark Bowman

**Comment by raleighlittles** [comment]
Mark, thanks for taking the time to review the PR -- I'm a bit embarrassed that I didn't manage to catch those 2 errors you mentioned. Last-minute debugging while you're tired is never a good recipe for success, apparently.

**Comment by markBowman** [comment]
You're welcome. Don't worry about the bugs. We understand you were doing this on top of your usual workload.

**Pull request closed without merging**